src/scene/skybox/sky.js

```diff
@@ -59,6 +75,19 @@ class Sky {
         this.projectedSkydomeCenterId = this.device.scope.resolve('projectedSkydomeCenter');
     }
 
+
+    applySettings(render) {
+        if (render.skyPosition) {
```

Comment:
scale and position should be set on the this.node instead of new properties, the same way the example does it:
```
this.node.setLocalScale(new pc.Vec3(data.get('data.skybox.scale') ?? [1, 1, 1]));
this.node.setLocalPosition(new pc.Vec3(data.get('data.skybox.position') ?? [0, 0, 0]));
```

and the added `position` and `scale` properties are not needed.

Reply:
Is it too late to reconsider exposing the `node` property? Maybe `scale`, `position` and `rotation` properties would be kinda nice. Then we could just alias `Scene#skyboxRotation` to `Sky#rotation`.

Reply:
Okay yea I was wondering if I was going to set them or leave that bit to you

Reply:
we need both rotations though. The node positions / rotates / scales the mesh itself.
but scene.skyboxRotation rotates the texture.

Reply:
I don't understand why the texture can't also be rotated with a quaternion.

Reply:
Maybe alias `Scene#skyboxRotation` to `Scene#skyTextureRotation` for more clarity then?

Reply:
maybe in a separate PR. I'd avoid doing it in this PR, this one should focus on the task that is needed.